### PR TITLE
[10.0][mig]-sale stock operating unit

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Operating Unit in Sales
=======================

This module was written to extend the Sales capabilities of Odoo.
This module introduces the operating unit to the Sales Order.
Security rules are defined to ensure that users can only display the
Sales Orders in which they are allowed access to.

Configuration
=============

Go to 'Settings / Technical / Actions / User-defined Defaults' and remove
the default set for the Shop.

### CODE REVIEW

File: sale_stock_operating_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+=======================
+Operating Unit in Sales
+=======================
```
Comment (darshan-serpent) [inline sale_stock_operating_unit/README.rst]: Sales Stock

File: sale_stock_operating_unit/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+This module was written to extend the Sales capabilities of Odoo.
+This module introduces the operating unit to the Sales Order.
+Security rules are defined to ensure that users can only display the
+Sales Orders in which they are allowed access to.
+
```
Comment (darshan-serpent) [inline sale_stock_operating_unit/README.rst]: module description does not matches

**Review by serpentcs-dev1 — APPROVED** [review]

File: sale_stock_operating_unit/README.rst (hunk 1 above)
Comment (serpentcs-dev1) [inline sale_stock_operating_unit/README.rst]: sale stock

File: sale_stock_operating_unit/README.rst (hunk 2 above)
Comment (serpentcs-dev1) [inline sale_stock_operating_unit/README.rst]: description does not match

Comment (LoisRForgeFlow) [reply sale_stock_operating_unit/README.rst]: You're right, I'll try to fix that.

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request merged**